xarray/tests/test_dataarray.py

```diff
@@ -3435,7 +3448,7 @@ def test_rolling_count_correct():
     result = da.rolling(time=11, min_periods=None).count()
     expected = DataArray(
         [np.nan, np.nan, np.nan, np.nan, np.nan, np.nan,
-         np.nan, np.nan, np.nan, np.nan, 8], dims='time')
+         np.nan, np.nan, np.nan, np.nan, np.nan], dims='time')
```

Comment:
I think the last element should be `np.nan` rather than 8, because `8 < min_periods=11`.